contracts/prize-strategy/multiple-winners/MultipleWinners.sol

```diff
@@ -1,12 +1,19 @@
 // SPDX-License-Identifier: MIT
-
 pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
 
 import "../PeriodicPrizeStrategy.sol";
 
 contract MultipleWinners is PeriodicPrizeStrategy {
 
+  struct MultipleWinnersPrizeSplit {
+      address target;
+      uint8 percentage;
```

Comment:
should this type not be larger than uint8 to support the max of 1000?

Reply:
Good catch - originally went with 100 and changed last night realizing we might want some single decimal precision.